.github/workflows/scheduled_ci.yml

```diff
@@ -0,0 +1,21 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+name: scheduled-ci
+on:
+  schedule:
+    - cron: 30 0,12 * * *
```

Comment:
Would this run for every fork? We should document the side effects of forking on CI runs.

Reply:
It only runs if the workflow is enabled.  That's one of the reasons to separate the scheduled workflow, so that it can be kept disabled in forks, but still have regular CI on push.

Reply:
> separate the scheduled workflow, so that it can be kept disabled in forks

The `schedule-ci `is active and triggered with the `schedule` event that occurs every 12 hours, right?  How are we going to disable it (the workflow) for the new contributor forks?